app/generator/conversion.js

```diff
@@ -0,0 +1,9 @@
+const CONVERSION_RATE = 2.835
+
+const millimetresToPoints = (millimetres) => {
+  return millimetres * CONVERSION_RATE
+}
```

Comment:
GOV.UK Notify has strict requirements for margins and placement of the address.  All of those measurements are in millimetres.  `pdfmake` works in points, so have added this to convert mm to points.

To validate I have also measured the placement of address and margins to ensure they are "safe" using Adobe acrobat's measuring tools.

Letter specs -> https://www.notifications.service.gov.uk/using-notify/guidance/letter-specification